jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/kafka/topics/RawEncodedJsonTopics.java

```diff
@@ -4,15 +4,19 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+/**
+ * Configuration class for raw encoded JSON Kafka topics. Prefix: ode.kafka.topics.raw-encoded-json
```

Comment:
suggestion(non-blocking): since the prefix is already available on line 11 below, we should remove the duplicated knowledge to follow the DRY principle and prevent accidental drift
```suggestion
 * Configuration class for raw encoded JSON Kafka topics.
```

Reply:
addressed